test/expect.js

```diff
@@ -454,53 +454,53 @@ describe('expect', function () {
 
   it('within(start, finish)', function(){
     expect(5).to.be.within(5, 10);
-    expect(5).to.be.within(3,6);
-    expect(5).to.be.within(3,5);
-    expect(5).to.not.be.within(1,3);
-    expect('foo').to.have.length.within(2,4);
-    expect('foo').to.have.lengthOf.within(2,4);
-    expect([ 1, 2, 3 ]).to.have.length.within(2,4);
-    expect([ 1, 2, 3 ]).to.have.lengthOf.within(2,4);
+    expect(5).to.be.within(3, 6);
+    expect(5).to.be.within(3, 5);
+    expect(5).to.not.be.within(1, 3);
+    expect('foo').to.have.length.within(2, 4);
+    expect('foo').to.have.lengthOf.within(2, 4);
+    expect([ 1, 2, 3 ]).to.have.length.within(2, 4);
+    expect([ 1, 2, 3 ]).to.have.lengthOf.within(2, 4);
 
     err(function(){
-      expect(5).to.not.be.within(4,6, 'blah');
+      expect(5).to.not.be.within(4, 6, 'blah');
     }, "blah: expected 5 to not be within 4..6");
 
     err(function(){
-      expect(5, 'blah').to.not.be.within(4,6);
+      expect(5, 'blah').to.not.be.within(4, 6);
     }, "blah: expected 5 to not be within 4..6");
 
     err(function(){
-      expect(10).to.be.within(50,100, 'blah');
+      expect(10).to.be.within(50, 100, 'blah');
     }, "blah: expected 10 to be within 50..100");
 
     err(function () {
-      expect('foo').to.have.length.within(5,7, 'blah');
+      expect('foo').to.have.length.within(5, 7, 'blah');
     }, "blah: expected \'foo\' to have a length within 5..7");
 
     err(function () {
-      expect('foo', 'blah').to.have.length.within(5,7);
+      expect('foo', 'blah').to.have.length.within(5, 7);
     }, "blah: expected \'foo\' to have a length within 5..7");
 
     err(function () {
-      expect('foo').to.have.lengthOf.within(5,7, 'blah');
+      expect('foo').to.have.lengthOf.within(5, 7, 'blah');
     }, "blah: expected \'foo\' to have a length within 5..7");
 
     err(function () {
-      expect([ 1, 2, 3 ]).to.have.length.within(5,7, 'blah');
+      expect([ 1, 2, 3 ]).to.have.length.within(5, 7, 'blah');
     }, "blah: expected [ 1, 2, 3 ] to have a length within 5..7");
 
     err(function () {
-      expect([ 1, 2, 3 ]).to.have.lengthOf.within(5,7, 'blah');
+      expect([ 1, 2, 3 ]).to.have.lengthOf.within(5, 7, 'blah');
     }, "blah: expected [ 1, 2, 3 ] to have a length within 5..7");
 
     err(function () {
       expect(null).to.be.within(0, 1, 'blah');
-    }, "blah: expected null to be a number");
+    }, "blah: the arguments to within must be numbers or dates");
```

Comment:
Yes this message is a little vague. Can we add an extra `if` to just throw saying something like "blah: expected null to be a number or date"